src/java/org/apache/cassandra/service/accord/AccordExecutorLoops.java

```diff
@@ -24,7 +24,7 @@
 import java.util.function.IntFunction;
 
 import accord.utils.Invariants;
-import org.agrona.collections.LongHashSet;
+import io.netty.util.collection.LongObjectHashMap;
```

Comment:
curious why you moved off `agrona` to `netty`'s.  Not against it, just wondering if there was a reason

Reply:
Mistake - intended only to switch to a map to avoid unlikely scenario of thread if being reused for a dead thread (close to impossible given how we use it, but safer). Will switch to Agrona map version for consistency